src/EditorFeatures/Core/Implementation/Classification/CopyPasteAndPrintingClassificationBufferTaggerProvider.Tagger.cs

```diff
@@ -45,17 +45,15 @@ private class Tagger : ForegroundThreadAffinitizedObject, IAccurateTagger<IClass
                 _owner = owner;
                 _subjectBuffer = subjectBuffer;
 
-                const TaggerDelay Delay = TaggerDelay.Short;
```

Comment:
this delay wasn't even relevant.  the copy/paste tagger doen't even wait any amount of time when it receives events.